django/forms/extras/__init__.py

```diff
@@ -1,3 +1,12 @@
-from django.forms.extras.widgets import SelectDateWidget
+import warnings
+
+from django.forms.widgets import SelectDateWidget  # NOQA
+from django.utils.deprecation import RemovedInDjango21Warning
 
 __all__ = ['SelectDateWidget']
+
+
+warnings.warn(
+    "django.forms.extras is deprecated. You can find "
+    "SelectDateWidget in django.forms.widgets instead.",
+    RemovedInDjango21Warning, stacklevel=2)
```

Comment:
I copied this snippet from https://github.com/django/django/blob/master/django/db/models/sql/aggregates.py#L16-L19,  I presume it should be fixed over there too?